xml/installation-installation-esx-prepare_deploy_esx_cloud_ovsvapp.xml

```diff
@@ -325,14 +325,14 @@ ansible-playbook -i hosts/verb_hosts ops-console-reconfigure.yml </screen>
     Change the directory. The <literal>cinder.conf.j2</literal> is present in
     following directories:
    </para>
-<screen>cd /home/stack/helion/hos/ansible/roles/_CND-CMN/templates</screen>
+<screen>cd ~/openstack/hos/ansible/roles/_CND-CMN/templates</screen>
```

Comment:
Should the `/hos/` still be in there? (I genuinely don't know the dir structure, so can't say, but since you did `/hos/`->`/ardana/` in #138, I was wondering.)

Reply:
More information requested in the originating bug request. No answer as of UTC 20:15. Might be good to incorporate these changes and pick up the filename changes later.